BaikalDataGenerator/Source/render.cpp

```diff
@@ -99,52 +105,51 @@ Render::Render(const std::string &file_name,
     m_renderer = m_factory->CreateRenderer(Baikal::ClwRenderFactory::RendererType::kUnidirectionalPathTracer);
     m_controller = m_factory->CreateSceneController();
 
-    for (const auto output_info: outputs_collection)
+    for (auto& output_info: outputs_collection)
     {
         m_outputs.push_back(m_factory->CreateOutput(output_width, output_height));
         m_renderer->SetOutput(output_info.type, m_outputs.back().get());
+        output_info.width = output_width;
+        output_info.height = output_height;
     }
 
+    std::replace(file_name.begin(), file_name.end(), '\\', '/');
```

Comment:
I think it should be done only for Windows. Linux files can contain backslashes in names

Reply:
removed